docs/src/selectors.md

```diff
@@ -755,6 +755,7 @@ In react selectors, component names are transcribed with **CamelCase**.
 Selector examples:
 
 - match by **component**: `_react=BookItem`
+- match by component and **key**: `_react=BookItem[key = '2']`
```

Comment:
@aslushnikov / @kblok For the docs, should the `key` example go last (and/or come with a note/caveat linking to https://playwright.dev/docs/selectors#best-practices or mentioning React selector best practices)?

In most cases, I think the other React selector examples here are preferable (alone or chained with Playwright's other selectors) as they will be less fragile than `key` (at least anecdotally thinking through how often I've changed `key`s vs. other props like e.g. `author`.)

Reply:
fair. Feel free to amend!